systemtest/src/main/java/io/strimzi/systemtest/resources/operator/HelmInstallation.java

```diff
@@ -0,0 +1,134 @@
+            Matcher matcher = StUtils.IMAGE_PATTERN_FULL_PATH.matcher(Environment.BRIDGE_IMAGE);
+
+            // in case that the image has correct value, we can configure the values
+            if (matcher.matches()) {
+                bridgeRegistry = Optional.ofNullable(matcher.group("registry")).orElse(bridgeRegistry);
+                bridgeRepository = Optional.ofNullable(matcher.group("org")).orElse(bridgeRepository);
+                bridgeName = Optional.ofNullable(matcher.group("image")).orElse(bridgeName);
+                bridgeTag = Optional.ofNullable(matcher.group("tag")).orElse(bridgeTag);
+            }
+        }
+
+        installCommand.set("kafkaBridge.image.registry", bridgeRegistry);
+        installCommand.set("kafkaBridge.image.repository", bridgeRepository);
+        installCommand.set("kafkaBridge.image.name", bridgeName);
+        installCommand.set("kafkaBridge.image.tag", bridgeTag);
+
+        // Additional config
+        installCommand.set("image.imagePullPolicy", Environment.OPERATOR_IMAGE_PULL_POLICY);
+        installCommand.set("resources.requests.memory", TestConstants.CO_REQUESTS_MEMORY);
+        installCommand.set("resources.requests.cpu", TestConstants.CO_REQUESTS_CPU);
+        installCommand.set("resources.limits.memory", TestConstants.CO_LIMITS_MEMORY);
+        installCommand.set("resources.limits.cpu", TestConstants.CO_LIMITS_CPU);
+        installCommand.set("logLevelOverride", Environment.STRIMZI_LOG_LEVEL);
+        installCommand.set("fullReconciliationIntervalMs", Long.toString(clusterOperatorConfiguration.getReconciliationInterval()));
+        installCommand.set("operationTimeoutMs", Long.toString(clusterOperatorConfiguration.getOperationTimeout()));
+        // As FG is CSV, we need to escape commas for interpretation of helm installation string
+        installCommand.set("featureGates", Environment.STRIMZI_FEATURE_GATES.replaceAll(",", "\\\\,"));
+        installCommand.set("watchAnyNamespace", clusterOperatorConfiguration.isWatchingAllNamespaces());
+        installCommand.set("replicas", clusterOperatorConfiguration.getReplicas());
+
+        if (!clusterOperatorConfiguration.isWatchingAllNamespaces() && !clusterOperatorConfiguration.getNamespacesToWatch().equals(clusterOperatorConfiguration.getNamespaceName())) {
+            installCommand.set("watchNamespaces", buildWatchNamespaces());
+        }
+
+        // Set extraEnvVars into Helm Chart
+        if (clusterOperatorConfiguration.getExtraEnvVars() != null) {
+            int envVarIndex = 0;
+            for (EnvVar envVar: clusterOperatorConfiguration.getExtraEnvVars()) {
+                installCommand.set("extraEnvs[" + envVarIndex + "].name", envVar.getName());
+                installCommand.set("extraEnvs[" + envVarIndex + "].value", envVar.getValue());
+                envVarIndex++;
+            }
+        }
+
+
+        installCommand.call();
+    }
+
+    @Override
+    public void delete() {
```

Comment:
Why it is not handled automatically by RM as Olm and Yaml?

Reply:
because we are using the Helm client for installation, so what should be handled by RM?